test/microapps-ui-xcm/microapps-ui-main.liq.pools.test.ts

```diff
@@ -386,6 +386,45 @@ describe("Miocroapps UI liq pools tests", () => {
     expect(isEstRewardDisplayed).toBeTruthy();
   });
 
+  it("All promoted pools exist in app", async () => {
+    const sdk = await getMangataInstance();
+    const api = await getApi();
+    const promotedPoolsRewards =
+      await api.query.proofOfStake.promotedPoolRewards();
+    const promotedPools = JSON.parse(
+      JSON.stringify(Object.keys(promotedPoolsRewards.toHuman())),
+    );
+    const promotedPoolsLength = promotedPools.length;
+    const poolsInfo = [];
+    for (let i = 0; i < promotedPoolsLength; i++) {
+      const poolName = await sdk.query.getLiquidityPool(promotedPools[i]);
+      const firstTokenId = await sdk.query.getTokenInfo(poolName[0]);
+      const secondTokenId = await sdk.query.getTokenInfo(poolName[1]);
+      const a = {
+        poolID: promotedPools[i],
+        firstToken: firstTokenId.symbol,
+        secondToken: secondTokenId.symbol,
+      };
+      poolsInfo.push(a);
+    }
+    const poolsInfoLength = poolsInfo.length;
+
+    const sidebar = new Sidebar(driver);
+    await sidebar.clickNavLiqPools();
+    const poolsList = new LiqPools(driver);
+
+    for (let i = 0; i < poolsInfoLength; i++) {
+      const isPoolVisible =
+        (await poolsList.isPoolItemDisplayed(
+          "-" + poolsInfo[i].secondToken + "-" + poolsInfo[i].firstToken,
+        )) ||
+        (await poolsList.isPoolItemDisplayed(
+          "-" + poolsInfo[i].firstToken + "-" + poolsInfo[i].secondToken,
+        ));
+      expect(isPoolVisible).toBeTruthy();
+    }
+  });
```

Comment:
Plz validate that the length of the lists is equal, otherwise, we do not know if we are skipping some pools.

Reply:
Done